XML/TestCases/PerformanceTests-DPDK.xml

```diff
@@ -86,6 +90,111 @@
       <param>TEST_DURATION=DPDK_TEST_DURATION_IN_SECONDS</param>
       <param>MODES="rxonly io"</param>
       <param>CORES=DPDK_PERF_CORES</param>
+      <param>pmd=failsafe</param>
+    </TestParameters>
+    <Platform>Azure</Platform>
+    <Category>Performance</Category>
+    <Area>DPDK</Area>
+    <Tags>dpdk,dpdk-core,network,hv_netvsc,sriov</Tags>
+    <Priority>2</Priority>
+    <SetupConfig>
+      <Networking>SRIOV</Networking>
+      <SetupType>TwoVM2NIC</SetupType>
+      <OverrideVMSize>Standard_F32s_v2</OverrideVMSize>
+    </SetupConfig>
+  </test>
+  <test>
+    <testName>PERF-DPDK-FWD-PPS-DS15-NETVSCPMD</testName>
+    <testScript>DPDK-TESTCASE-DRIVER.ps1</testScript>
+    <files>.\Testscripts\Linux\dpdk-testFWD.sh,.\Testscripts\Windows\dpdk-testFWD.ps1</files>
+    <TestParameters>
+      <param>dpdkSrcLink="DPDK_SOURCE_URL"</param>
+      <param>TEST_DURATION=DPDK_TEST_DURATION_IN_SECONDS</param>
+      <param>CORES=(2 4 8)</param>
+      <param>pmd=netvsc</param>
+    </TestParameters>
+    <Platform>Azure</Platform>
+    <Category>Performance</Category>
+    <Area>DPDK</Area>
+    <Tags>dpdk,dpdk-core,network,hv_netvsc,sriov</Tags>
+    <Priority>2</Priority>
+    <SetupConfig>
+      <Networking>SRIOV</Networking>
+      <SetupType>ThreeVM2NIC</SetupType>
+    </SetupConfig>
+  </test>
+  <test>
+    <testName>PERF-DPDK-SINGLE-CORE-PPS-DS4-NETVSCPMD</testName>
```

Comment:
Yes. But these testcases are for checking DPDK performance with different VM sizes. We have same testcases with failsafe PMD too.

Reply:
I am aware of your design in DPDK, but it is not important to keep doing the same way because we have done that way. If I understand your test design, your test cases are not available with different VM size then. Correct? What makes to put that restriction for the performance measurement? I request you remove the VM size restriction in the test case name.